pkg/pod/status_test.go

```diff
@@ -660,3 +660,45 @@ func TestSortTaskRunStepOrder(t *testing.T) {
 		t.Errorf("Unexpected step order (-want, +got): %s", d)
 	}
 }
+
+func TestSortContainerStatuses(t *testing.T) {
+	samplePod := corev1.Pod{
+		Status: corev1.PodStatus{
+			ContainerStatuses: []corev1.ContainerStatus{
+				{
+					Name: "hello",
+					State: corev1.ContainerState{
+						Terminated: &corev1.ContainerStateTerminated{
+							FinishedAt: metav1.Time{Time: time.Now()},
+						},
```

Comment:
NIT: I think it would be nicer to invoke time.Now() only once; but it's quite unlikely that 5s pass between here and L679, so it's ok